ITEM.tsv

```diff
@@ -4086,7 +4086,7 @@ ITEM_20150323_004085	{memo X}
 ITEM_20150323_004086	Sweet Herb 
 ITEM_20150323_004087	{memo X} 
 ITEM_20150323_004088	Nutritious Tonic 
-ITEM_20150323_004089	A nutritious tonic made by diluting the concentrate that the prince gave you with Scorpio Venom. 
+ITEM_20150323_004089	A nutritious tonic made by diluting the concentrate that Mercenary Stephas gave you with Scorpio Venom. 
```

Comment:
Oh, then the whole line is a bit weird.  Should it say "the concentrate from the baron with Scorpio Venom"

Because the baron originally gives the concentrate to Stephas, and then Stephas gives it to you.